scripts/check-prometheusrules.py

```
@@ -0,0 +1,369 @@
+                path = Path(root, entry)
+                if path.suffix not in ('.yml', '.yaml'):
+                    continue
+                try:
+                    with path.open() as f:
+                        doc = yaml.safe_load(f)
+                    if (isinstance(doc, dict)
+                            and doc.get('kind') == 'PrometheusRule'
+                            and doc.get('apiVersion') == 'monitoring.coreos.com/v1'):
+                        files.append((path, doc))
+                except Exception:
+                    continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)